cli/command/inspect/inspector.go

```diff
@@ -84,6 +87,37 @@ func Inspect(out io.Writer, references []string, tmplStr string, getRef GetRefFu
 	return nil
 }
 
+// All fetches all objects using GetAllFunc and writes the json
+// representation to the output writer.
+func All(out io.Writer, tmplStr string, getAll GetAllFunc) error {
```

Comment:
I don't really get why this needs to use a callback to fetch the data? 

It pushes the data retrieval and error handling into this func which returns the error to the caller who has to handle it anyway, and forces the caller to provide a func that returns an error which may not be required.

It's a little simpler (and the usual way of doing things) if this func accepts the data directly, and leave data retrieval and error handling to the caller:

```suggestion
func All(out io.Writer, tmplStr string, elements []interface{}) error {
```

Reply:
done